### PR TITLE
Update slides for WarsawJS Workshop #41

### PR DESCRIPTION
**Pull request by malyfko**

### CODE REVIEW

File: slides.md [hunk 1]
```diff
@@ -30,11 +30,12 @@ class: middle, center, slide-card-list
 
 ]
 
-.card[![](./images/avatars-volunteers/anna-lukianova.jpg) Anna Lukianova]
```
Comment (piecioshka) [inline slides.md]: Ania jednak będzie :)

File: slides.md [hunk 2]
```diff
@@ -303,6 +305,7 @@ class: middle
 ---
 
 
+
```
Comment (piecioshka) [inline slides.md]: ups :)

Comment (malyfko) [reply slides.md]: @piecioshka fixed ;)

**Comment by piecioshka** [comment]
Thanks @malyfko 🎊

**Pull request merged**